cmd/syncer/main.go

```diff
@@ -118,19 +118,29 @@ func main() {
 		os.Exit(1)
 	}
 
-	err = startSyncers(ctrl.SetupSignalHandler(), syncedResources, client.ObjectKey{Namespace: controlPlaneConfigSecretNamespace, Name: controlPlaneConfigSecretName}, mgr)
+	syncerContext, cancel := context.WithCancel(context.Background())
+	defer func() {
+		cancel()
+	}()
```

Comment:
```suggestion
	defer cancel()
```

I think this would do the same

Reply:
oh good spot, I'll update this.